test/pcaps/ikev2_nat_t/ikev2_decryption_table

```diff
@@ -0,0 +1,3 @@
+89922c915f35570e,98d56d32e2a04742,f49c0d53899c8aef4435e2850619825dacc1c8b4bfd636f7175032cfbe4c57db8b7b260f,42e53af4eabccaabf10ad37da3b980edc5af5a4c742fd47a29b4341f7847caf92ccacf3d,"AES-GCM-256 with 16 octet ICV [RFC5282]",,,"NONE [RFC4306]"
```

Comment:
Do you use this? Otherwise please remove ig

Reply:
Removed.